### ISSUE TITLE
Perform accessibility improvements

### ISSUE THREAD

**Issue by benheet** [ndlibersa/resources#14]
WTAMU used the Firefox accessibility extension to identify several accessibility failures that they have asked us to address. This issue should be explored as time permits.
